src/db/schema.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    AGE_BRACKETS,
+    DEGREES,
+    GENDERS,
+    STUDENT_STATUSES,
+    STUDENT_TYPES,
+} from '@/constants/student-info';
+import { sql } from 'drizzle-orm';
+import { integer, sqliteTable, text } from 'drizzle-orm/sqlite-core';
+import { nanoid } from 'nanoid';
+
+export const members = sqliteTable('members', {
+    id: text('id')
+        .$defaultFn(() => nanoid())
+        .primaryKey(),
+
+    clerkId: text('clerk_id').notNull(),
+    email: text('email').notNull(),
+    firstName: text('first_name').notNull(),
+    lastName: text('last_name').notNull(),
+
+    studentStatus: text('student_status', { enum: STUDENT_STATUSES }).notNull(),
+    studentId: text('student_id'),
+    gender: text('gender', { enum: GENDERS }).notNull(),
+    ageBracket: text('age_bracket', { enum: AGE_BRACKETS }).notNull(),
+    degree: text('degree', { enum: DEGREES }),
+    studentType: text('student_type', { enum: STUDENT_TYPES }),
+
+    emailPreferences: text('email_preferences', { mode: 'json' }),
+
+    membershipExpiredAt: integer('membership_expired_at', { mode: 'timestamp' }),
```

Comment:
Change to `membershipExpiresAt` and `membership_expires_at` as membership could still be ongoing (it doesn't really make sense to say "expired" if it hasn't, if you get what I mean).

Reply:
I really need to learn more English 🤣